cmd/dep/init.go

```diff
@@ -238,14 +181,19 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 	return nil
 }
 
-// contains checks if a array of strings contains a value
-func contains(a []string, b string) bool {
-	for _, v := range a {
-		if b == v {
-			return true
-		}
+func getDirectDependencies(root, cpr string) (*pkgtree.PackageTree, map[string]bool, error) {
```

Comment:
Do we actually need to return a `pkgtree.PackageTree` pointer here? If not, let's return a value for consistency.